rollbar/project_access_tokens.go

```diff
@@ -27,22 +26,12 @@ type listProjectAccessTokensResponse struct {
 func (c *Client) ListProjectAccessTokens(projectID int) ([]*ProjectAccessToken, error) {
 	var data listProjectAccessTokensResponse
 
-	url := fmt.Sprintf("%sproject/%d/access_tokens/?access_token=%s", c.APIBaseURL, projectID, c.APIKey)
-
-	req, err := http.NewRequest("GET", url, nil)
-
-	if err != nil {
-		return nil, err
-	}
-
-	bytes, err := c.makeRequest(req)
-
+	bytes, err := c.get("project", strconv.Itoa(projectID), "access_tokens")
```

Comment:
wouldn't `string(projectID)` be simpler?

Reply:
@graffzon `string(23)` will result in the unicode of value 23.